frame/contracts/src/wasm/runtime.rs

```diff
@@ -813,6 +818,27 @@ define_env!(Env, <E: Ext>,
 		ctx.set_storage(key_ptr, value_ptr, value_len).map(|_| ())
 	},
 
+	// Replace contract code.
+	//
+	// # Parameters
+	//
+	// - code_hash_ptr: a pointer to the buffer that contains new contract code
+	//
+	// # Errors
+	//
+	// `ReturnCode::CodeNotFound`
+	[__unstable__] seal_set_code_hash(ctx, code_hash_ptr: u32) -> ReturnCode => {
```

Comment:
Please move it to the bottom of the macro invocation. It doesn't really fit between the different `set_storage` versions.

Reply:
@yarikbratashchuk can you address this?

Reply:
@yarikbratashchuk can you please move this, or justify why you're leaving it in-between the different `set_storage` versions?